bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -72,6 +72,7 @@ class Presentation extends PureComponent {
       zoom: 100,
       fitToWidth: false,
       isFullscreen: false,
+      isUnmounted: false,
```

Comment:
Instead of using a state variable, consider using `Session` package and setting a variable on it.

Something like:
`Session.set('presentationComponentUnmounted', false);`